src/ansys/grantami/bomanalytics/_query_results.py

```diff
@@ -766,26 +763,20 @@ class BomSustainabilityQueryResult(ResultBaseClass):
 
     def __init__(
         self,
-        results: List[models.GetSustainabilityForBom2301Response],
+        results: List[models.GetSustainabilityForBomResponse],
         messages: List[models.CommonLogEntry],
     ) -> None:
         super().__init__(messages)
         self._response = results[0]
-        if not self._response.parts:
+        if not self._response.part:
             raise ValueError(
                 "Found no part in BoM sustainability response. Ensure the request BoM defines a single root part."
             )
-        if len(self._response.parts) > 1:
-            warnings.warn(
-                f"BomSustainabilityQuery only supports a single root part (found {len(self._response.parts)}). "
-                f"Additional root parts do not include sustainability results and are not exposed in the query result"
-                f" properties."
-            )
         # Exposing only a single root part:
         # API V1 only processes the first root part but still returns part empty part objects for extra root parts.
```

Comment:
Do we still need this comment?

Reply:
Good point, probably not. V2 does what we would expect, so both comments can be removed.